cityofaustin/atd-airflow
# Trail Counters to Production

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/12453

## Associated repo
https://github.com/cityofaustin/atd-trail-counter-data

## Testing

**Steps to test:**
1. git pull this branch down to your local machine
2. `docker compose build` this version of the airflow stack
3.  `docker compose run --rm airflow-cli dags test atd_trail_counters` will run the DAG
4. View the logs and verify it's pulling data from the ecocounter website something like:
```
DEBUG:counter_data.py:0 records found for Shoal Creek Blvd at 38th St PC
DEBUG:counter_data.py:0 records found for Southern Walnut Creek Trail - Urban Trail
...
```


---
#### Ship list
